QualityControl/Dockerfile

```diff
@@ -16,17 +21,38 @@ RUN apk add --no-cache \
     giflib-dev \
     musl-dev
 
-# Working directory
-WORKDIR /usr/src/app
+# Export port
+EXPOSE 8080
 
-# Install dependencies
-COPY . .
+FROM base AS dependencies
+
+# Install dependencies early for Docker layer caching
+COPY package*.json ./
 RUN npm ci --silent
-RUN npm i -g nodemon --save-dev
```

Comment:
You removed the `nodemon` installation which now makes it impossible to correctly run the `docker-dev` command on a fresh installation